frontend/src/pages/PersonalMaengelList.js

```diff
@@ -36,7 +36,7 @@ export default function PersonalMaengelList({ user, token, ...props }) {
           {mangelList.length > 0 && (
             <ul>
               {mangelList.map((mangel, index) => {
-                return <ListItem key={index} mangel={mangel} />
+                return <ListItem id={index} mangel={mangel} />
```

Comment:
Egal ob mit key oder id, ich bekomme beim ersten Laden der Seite immer eine Fehlermeldung. Danach nicht mehr. Ich hab noch nicht verstanden warum.
`Warning: ListItem: `key` is not a prop. Trying to access it will result in `undefined` being returned. If you need to access the same value within the child component, you should pass it as a different prop`